src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/video-unit-form/video-unit-form.component.ts

```diff
@@ -21,27 +21,36 @@ function isVideoOnlyTumUrl(url: URL): boolean {
     return url?.searchParams.get('video_only') === '1';
 }
 
-function videoSourceTransformUrlValidator(control: AbstractControl): ValidationErrors | null {
-    const invalidVideoUrlError = { invalidVideoUrl: true };
+function videoSourceTransformUrlValidator(control: AbstractControl): ValidationErrors | undefined {
     const urlValue = control.value;
-    if (!urlValue) return null;
+    if (!urlValue) {
+        return undefined;
+    }
+    let parsedUrl, url;
     try {
-        return isTumLiveUrl(new URL(urlValue)) || urlParser.parse(urlValue) ? null : invalidVideoUrlError;
+        url = new URL(urlValue);
+        parsedUrl = urlParser.parse(urlValue);
     } catch {
-        return invalidVideoUrlError;
+        //intentionally empty
+    }
+    // The URL is valid if it's a TUM-Live URL or if it can be parsed by the js-video-url-parser.
+    if ((url && isTumLiveUrl(url)) || parsedUrl) {
+        return undefined;
     }
+    return { invalidVideoUrl: true };
 }
 
-function videoSourceUrlValidator(control: AbstractControl): ValidationErrors | null {
-    const invalidVideoUrlError = { invalidVideoUrl: true };
+function videoSourceUrlValidator(control: AbstractControl): ValidationErrors | undefined {
+    let url;
     try {
-        const urlValue = control.value;
-        const url = new URL(urlValue);
-        if (isTumLiveUrl(url)) return isVideoOnlyTumUrl(url) ? null : invalidVideoUrlError;
-        return urlParser.parse(urlValue) ? invalidVideoUrlError : null;
+        url = new URL(control.value);
     } catch {
-        return invalidVideoUrlError;
+        // intentionally empty
     }
+    if (url && !(isTumLiveUrl(url) && !isVideoOnlyTumUrl(url))) {
```

Comment:
- we do not check the `urlParser.parse(urlValue)` anymore
- the if also correct if the URL is not from TumLive.

Are those things on purpose?